Feature/add links to footer

positiveimpact opened the pull request:
I added the "impressum" link as well as social media links to the footer.
Additionally I made the footer always stay at the bottom of the page. Before it wasn't of the bottom if the height of the content of the page was smaller than the screen height. 
To accomplish this I had to add an additional div to layout.js and AboutLayout.js. Since every layout needs this div, I decided to create the file LayoutWrapper.js which is used by both layout files. This way we only have to make changes in one place if we want to make any changes to that div in the future.

EvanHahn commented on public/data/projects.json:
```diff
@@ -5,7 +5,7 @@
       "image": "/images/canteen.jpg",
       "organisation_name": "sneep Erlangen e.V.",
-      "organisation_image": "/images/sneep.png",
+      "organisation_image": "images/sneep.png",
```
Should this be reverted?

positiveimpact replied:
Oh got I keep screwing this up for some reason. Reverted it.

EvanHahn commented on public/icons/facebook.svg:
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Should we use the [Material UI icon](https://material-ui.com/components/material-icons/) for Facebook instead? The same comment applies for the other icons—Twitter, GitHub, Instagram, etc.

positiveimpact replied:
Makes sense. Switched to Material UI Icons

EvanHahn commented on src/components/general/Footer.js:
```diff
@@ -1,22 +1,90 @@
 
 const useStyles = makeStyles(theme => ({
-  root: {
+  footer: {
```
Why did this name change?

positiveimpact replied:
I built a wrapper around it, removed it again and forgot to change it back. Reverted.

EvanHahn commented on src/components/layouts/AboutLayout.js:
```diff
@@ -2,32 +2,28 @@ import React from "react";
-    </>
+    <LayoutWrapper theme={aboutTheme}>
+      <Header />
+      <Container maxWidth={false} component="main" className={`${classes.main}`}>
```
Nitpick: could this just be `className={classes.main}`?

EvanHahn commented on src/components/layouts/LayoutWrapper.js:
```diff
@@ -0,0 +1,26 @@
+  leaveSpaceForFooter: {
+    position: "relative",
+    paddingBottom: 68,
+    minHeight: "100vh"
```
Nice!

EvanHahn commented on src/components/layouts/LayoutWrapper.js:
```diff
@@ -0,0 +1,26 @@
+const useStyles = makeStyles({
+  leaveSpaceForFooter: {
+    position: "relative",
+    paddingBottom: 68,
```
Should this use `theme`?

positiveimpact commented:
Thanks for the review! Implemented all suggested changes:
-added the "/" back in in organisation_image
-switched to material UI icons
-reverted className of root item to "root"
-Fixed the Nitpick
-used theme.spacing for height of footer and padding of body

I also removed some unused CSS that I found in Footer.js

Are there any other remarks? Otherwise I think this is ready to merge.
